README.md

```diff
@@ -1,2 +1,46 @@
 # m02-export-vk-comments
 Exports comments of a given club posts. Runs as a docker container
+
+## How to install
+
+Clone the repository and go the directory.
+
+    git clone git@github.com:dimdasci/m02-export-vk-comments.git
```

Comment:
Could make it a bit pretty - use markdown \```some code \``` for code

Reply:
@PhySci thank for suggestions!

- [x] print was replaced by logging
- [x] pandas `to_csv` was replaced by csv module, totally agree that just writing csv with pandas is super inefficient 

I haven't  changed readme yet. [Markdown says](https://www.markdownguide.org/basic-syntax/#code-blocks): 

> To create code blocks, indent every line of the block by at least four spaces or one tab

Why do you think it's a good idea to use ```some code ``` for code instead?

Reply:
I mean use code block in your example. Of course, ```some code``` should be replaced on your code ```git clone ...```

Reply:
It seems like misunderstanding as we both are talking about code block formatting ) 

I use indentation for code block formatting as recommended in the markdown documentation mentioned above. I believe the \` or \`\`\` are good for inline code formatting. 

In fact I see no visual difference between using ``` or indentation. I was wondering if you could distinguish which of the following block is formatted with indentation and which one with  \`\`\`?

    git clone ...
    or some other code

and 

```
git clone ...
or some other code
```

Reply:
I got it.
 Indentation is also fine, of course!

Reply:
cool, tnanx for review!